### ISSUE TITLE
Write five slide presentations for OpenDreamKit

### ISSUE THREAD

**Issue by nthiery** [OpenDreamKit/OpenDreamKit#157]
For developers: (draft)[http://opendreamkit.org/about-developers]

For mathematicians: (draft)[http://opendreamkit.org/about-mathematicians]

**Comment by JohnCremona** [comment]
I looked at the "for mathematicians" version:

 - The slideshow starts with 3 blank pages

- on page 1 I do not like "even for pure mathematicians".    Pure
mathematicians were always at the forefront of mathematical computation --
Turing computed zeros of zeta, Birch & Swinnerton0Dyer used computations to
formulate their conjecture in the 1950s, ....

so how about changing "even in pure mathematics" to "in pure mathematics
and applications, teaching an research" ?

John

**Comment by nthiery** [comment]
> The slideshow starts with 3 blank pages

Yup; let's not worry about it for now, it's a shortcoming of the slide-viewer we are currently using, and Luca will fix that.

>- on page 1 I do not like "even for pure mathematicians".    Pure
> mathematicians were always at the forefront of mathematical computation --
> Turing computed zeros of zeta, Birch & Swinnerton0Dyer used computations to
> formulate their conjecture in the 1950s, ....

Indeed! Yet, nowadays, there is a popular believe that hard core computational maths is essentially just numerical analysis, and I was trying to counterweight this. Obviously the current formulation does not work. Let's find something better.

**Comment by nthiery** [comment]
For now, I made the title of that section:

## Long standing and booming role of computers in pure mathematics

Further editing welcome.

**Comment by takluyver** [comment]
Relative links appear to be broken when you go into slideshow view, as they become relative to the slideshow viewer URL. I noticed this with 'We recruit' in the developer slideshow.

'We recruit' should possibly be 'We are recruiting' - the latter sounds more natural in English.

**Comment by nthiery** [comment]
@takluyver: Thanks for your comments. I fixed the English as you suggested. The relative links should be fixed when we switch to an inplace slideshow.

**Comment by nthiery** [comment]
I'd really would need help to improve the text for mathematicians.

@stevelinton (for example) could you reserve some time to work on them?

Thanks!
